sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -122,7 +122,7 @@ object functions {
    * @group agg_funcs
    * @since 1.3.0
    */
-  def sum(e: Column): Column = Sum(e.expr)
+  def sum(a: Column): Column = Sum(a.expr)
```

Comment:
actually after thinking about it more ... this breaks scala source compatibility. Do you mind switching back to "e" and just use "e" for everything? It might've meant "expression". 

For the new ones, I'd use "expr".